src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/controller/CsipAdminController.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.validation.Valid
+import org.springdoc.core.annotations.ParameterObject
+import org.springframework.http.HttpStatus
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.constant.ROLE_CSIP_UI
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.CsipSearchResults
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.FindCsipRequest
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.service.CsipSearchService
+import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
+
+@RestController
+@RequestMapping
+@Tag(name = "8. CSIP Admin Controller", description = "Endpoints for admin features such as search and reporting")
+class CsipAdminController(private val search: CsipSearchService) {
```

Comment:
While these might only get used by managerial staff, they aren't restricted to any role on the front end. It would be fine to put all of these search and reporting type endpoints in the CSIP record controller